manifests/init.pp

```diff
@@ -120,6 +120,10 @@
 #
 # $vhost_priority::             Defines Apache vhost priority for the Foreman vhost conf file.
 #
+# $server_port::                Defines Apache port for HTTP requests
```

Comment:
Can you add `type:integer` below these two parameters please? It's used by Kafo to determine the data type.